src/android/ExifHelper.java

```diff
@@ -72,6 +73,7 @@ public void createOutFile(String filePath) throws IOException {
     public void readExifData() {
         this.aperture = inFile.getAttribute(ExifInterface.TAG_APERTURE);
         this.datetime = inFile.getAttribute(ExifInterface.TAG_DATETIME);
+        this.datetimeOriginal = inFile.getAttribute(ExifInterface.TAG_DATETIME_ORIGINAL);
```

Comment:
What is the difference with this one and the datetime one?

Reply:
The DateTimeOriginal value doesn't get updated when the file is modified